spring-cloud-dataflow-docs/src/main/asciidoc/streams.adoc

```diff
@@ -208,33 +208,113 @@ $ tail -f /var/folders/wn/8jxm_tbd1vj28c8vj37n900m0000gn/T/spring-cloud-dataflow
+║                              │Cron Trigger.                 │                              │                              ║
+║trigger.initial-delay         │Initial delay for periodic    │0                             │java.lang.Integer             ║
+║                              │triggers.                     │                              │                              ║
+║trigger.max-messages          │Maximum messages per poll, -1 │1                             │java.lang.Long                ║
+║                              │means infinity.               │                              │                              ║
+║trigger.date-format           │Format for the date value.    │<none>                        │java.lang.String              ║
+╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝
 ----
 
-IMPORTANT: See <<spring-cloud-dataflow-stream-app-labels>>.
+Below is the white listed properties for the app `log`:
+
+[source,bash]
+----
+╔══════════════════════════════╤══════════════════════════════╤══════════════════════════════╤══════════════════════════════╗
+║         Option Name          │         Description          │           Default            │             Type             ║
+╠══════════════════════════════╪══════════════════════════════╪══════════════════════════════╪══════════════════════════════╣
+║log.name                      │The name of the logger to use.│<none>                        │java.lang.String              ║
+║log.level                     │The level at which to log     │<none>                        │org.springframework.integratio║
+║                              │messages.                     │                              │n.handler.LoggingHandler$Level║
+║log.expression                │A SpEL expression (against the│payload                       │java.lang.String              ║
+║                              │incoming message) to evaluate │                              │                              ║
+║                              │as the logged message.        │                              │                              ║
+╚══════════════════════════════╧══════════════════════════════╧══════════════════════════════╧══════════════════════════════╝
+----
+
+The application properties for `time` and `log` apps can be specified at the time of `stream` creation as follows:
 
-==== Passing stream partition properties
+[source,bash]
+----
+dataflow:> stream create --definition "time --fixed-delay=5 | log --level=WARN" --name ticktock
+----
+
+Note that the properties `fixed-delay` and `level` defined above for the apps `time` and `log` are the `short-form` property names provided by the shell completion.
+These `short-form` property names are applicable only for the white-listed properties and in all other cases, only `fully qualified` property names should be used.
```

Comment:
reqular quotes instead of backticks for "short-form" - and fully-qualified should also not have backticks (these are not keywords or code)... maybe italics in the latter case though